lib/framework/widget/camera_manager.dart

```diff
@@ -15,32 +15,44 @@ class CameraManager {
         ),
       );
     }
-    // if camera action is not null 
+    // if camera action is not null
     else {
-    // camera instance is created for set properties to setter
-    // if any other camera instance is created already automatically remove beacuse camera instance is dispose
-    // when screen is close
-     
+      // camera instance is created for set properties to setter
+      // if any other camera instance is created already automatically remove beacuse camera instance is dispose
+      // when screen is close
+
       CameraScreen camera = CameraScreen();
       cameraAction.options!['mode'] == null
           ? camera.setProperty('mode', CameraMode.both)
           : camera.setProperty('mode', c(cameraAction.options!['mode']));
       cameraAction.options!['initialCamera'] == null
           ? camera.setProperty('initialCamera', InitialCamera.back)
           : camera.setProperty(
-          'initialCamera', i(cameraAction.options!['initialCamera']));
+              'initialCamera', i(cameraAction.options!['initialCamera']));
       cameraAction.options!['useGallery'] == null
           ? camera.setProperty('useGallery', true)
           : camera.setProperty(
-          'useGallery', cameraAction.options!['useGallery']);
+              'useGallery', cameraAction.options!['useGallery']);
       cameraAction.options!['preview'] == null
           ? camera.setProperty('preview', false)
           : camera.setProperty('preview', cameraAction.options!['preview']);
       cameraAction.options!['maxCount'] == null
           ? camera.setProperty('maxCount', 1)
-          : camera.setProperty('maxCount', cameraAction.options!['maxCount']);      
+          : camera.setProperty('maxCount', cameraAction.options!['maxCount']);
+      cameraAction.options!['errormessage'] == null
+          ? ''
+          : camera.setProperty(
+              'errormessage', cameraAction.options!['errormessage']);
+      cameraAction.options!['imgPickerIcon'] == null
```

Comment:
@hamxa152 we are using image instead of img everywhere. Please be consistent and call is imagePickerIcon